lib/cli/src/Cardano/CLI.hs

```diff
@@ -1314,19 +1319,19 @@ tlsOption = TlsConfiguration
         <> help "The RSA Server key which signed the x.509 server certificate."
 
 smashURLOption
-    :: Parser URI
+    :: Parser PoolMetadataSource
 smashURLOption = option (eitherReader reader) $ mempty
-    <> long "smash-url"
-    <> metavar "URL"
-    <> help "Optional HTTP(S) address of a \
-            \Stakepool Metadata Aggregation Server."
+    <> long "pool-metadata-fetching"
+    <> metavar "STRATEGY"
+    <> value FetchNone
+    <> showDefaultWith showT
+    <> help "Pool Metadata fetching strategy. This setting will persist across restarts. Possible values: \
+            \ none, \
+            \ direct, \
```

Comment:
What's the difference between "none" and "direct"? :sweat_smile:

Reply:
I suppose: 
 - "none" - don't look for metadata at all
 - "direct" - look for metadata on chain
 - "uri"  - look in the SMASH server

I imagine "none" would be picked rather seldom. (Only case I can think is some maintenance/testing purposes or maybe exchanges running wallet and don't care about metadata)

Perhaps better default value would be "direct"? Which is also a current default...

Reply:
`none` was decided as the default in the ticket (ADP-427):

> About (1), the command-line option should be removed altogether and the wallet should instead wait for a strategy to be provided. Without strategy, no metadata will be fetched whatsover.

Reply:
As per: https://input-output-rnd.slack.com/archives/GBT05825V/p1602080513063300

If nothing is provided we should default to the latest setting in use.  If there was no previous setting in use, then it should default to "none".

Currently we always have "none" which erases any settings that were set in runtime.